Solutions/GitHub/Data Connectors/GithubWebhook/azuredeploy_GithubWebhook_API_FunctionApp.json

```diff
@@ -0,0 +1,189 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "FunctionName": {
+            "defaultValue": "GitHubWebhook",
```

Comment:
Please provide default value in lower case. Textbox validation fails for this value.
![image](https://user-images.githubusercontent.com/10944026/174075946-5c7fcc79-31dd-4673-a6cd-032a40fb3dc7.png)

Reply:
done